install/ui/util/make-css.sh

```diff
@@ -22,5 +22,5 @@ set -o errexit
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
 
-lesscpy -x $DIR/../less/ipa.less > $DIR/../css/ipa.css
+py3-lesscpy -x $DIR/../less/ipa.less > $DIR/../css/ipa.css
```

Comment:
See https://src.fedoraproject.org/rpms/python-lesscpy/pull-request/3

Reply:
@hroncok That's just F29 or are you planning to land the change in F28, too?

Reply:
Nope, sorry. That would be too disturbing for stable release. You can just BR and use `/usr/bin/lesscpy` if you want "single-spec".